src/cache/tracer_cache.jl

```diff
@@ -0,0 +1,34 @@
+using ClimaUtilities.ClimaArtifacts
+import ClimaUtilities.TimeVaryingInputs: TimeVaryingInput
+
+function tracer_cache(Y, atmos, prescribed_aerosol_names, start_date)
+    prescribed_aerosol_timevaryinginputs = (;)
+    prescribed_aerosol_fields = (;)
+
+    if !isempty(prescribed_aerosol_names)
+        prescribed_aerosol_names_as_symbols = Symbol.(prescribed_aerosol_names)
+        target_space = axes(Y.c)
+        timevaryinginputs = [
+            TimeVaryingInput(
+                joinpath(
+                    @clima_artifact("aerosol2005", ClimaComms.context(Y.c)),
+                    "aero_2005.nc",
```

Comment:
Would it be useful to add to the docs somewhere what is the expected structure of the aerosol file? We might want to swap the data files we are using now with something different.